core/src/main/java/org/infinispan/topology/TopologyManagementHelper.java

```diff
@@ -115,6 +115,7 @@ public void executeOnCoordinatorAsync(Transport transport, ReplicableCommand com
          if (trace)
             log.tracef("Attempting to execute command on self: %s", command);
          try {
+            // TODO can we remove this as it's always the ConfirmRebalancePhaseCommand?
```

Comment:
AFAIK, yes. I think can change the methods to receive `AbstractTopologyCommand` instead of `ReplicableCommand`.